README.md

```diff
@@ -109,7 +109,6 @@ Below are listed tentative CIPs still under discussion with the community. They
 | 73? | [Oracle Datum Standard](https://github.com/cardano-foundation/CIPs/pull/357) |
 | 76? | [Hash-Checked Data](https://github.com/cardano-foundation/CIPs/pull/363) |
 | 77? | [Verified Stake Pool Identity](https://github.com/cardano-foundation/CIPs/pull/361) |
-| 78? | [Extended Local Chain Sync Protocol](https://github.com/cardano-foundation/CIPs/pull/375) |
 | 79? | [Implement Ouroboros Leios to increase Cardano throughput](https://github.com/cardano-foundation/CIPs/pull/379) |
 | 80? | [Transaction Serialization Deprecation Cycle](https://github.com/cardano-foundation/CIPs/pull/372) |
 | 81? | [Tiered Pricing Protocol](https://github.com/cardano-foundation/CIPs/pull/381)
```

Comment:
```suggestion
```
(re-submitting from above review since I applied it to the wrong section of the text)

All other stalled proposals have been removed from the _Proposals Under Review_ list, whether they've been assigned a number or not.

Maybe this list should include any tentatively assigned numbers (e.g. 0081 for _Tiered Pricing Protocol_: information which we would lose if the redundant table entry were removed)... so interested visitors don't have to dig it out of the pull request.

Reply:
The follow-up question will be: for how long do we want to keep those around even after marking them as "stall"? In principle, this doesn't change the PR's link, so any past link to the proposal would still work.

Reply:
@KtorZ I would posit that the `Waiting for Author` tag on the PR is to get the author's attention, while appearance on the _Stalled_ list is to get the community's attention.  My feeling has always been we should close the PR when it appears (some time after tagging) that the author has lost interest... so likewise we should remove from _Stalled_ when it appears the community has lost interest. 🤔